File: HTML/META-INF/resources/frame/default/SideMenuForm.js
/**
 * メニューフォーム.
 */
class SideMenuForm extends MenuForm {
	/**
	 * ページの各エレメントとの対応付け.
	 */
	attach = function() {
		Form.prototype.attach.call(this);
		this.menu = this.newInstance(this.sideMenu);
		this.menu.init();
		this.menu.attach();
	}
}

File: src/dataforms/app/menu/page/SideMenuForm.java
package dataforms.app.menu.page;

import dataforms.menu.Menu;
import dataforms.menu.SideMenu;

/**
 * サイドメニューフォームクラス。
 *
 */
public class SideMenuForm extends MenuForm {
	/**
	 * コンストラクタ。
	 */
	public SideMenuForm() {
		super(null);
	}

	@Override
	protected Menu newMenuComponent() {
		SideMenu menu = new SideMenu();
		return menu;
	}


}
